### PR TITLE
Small tweaks while doing other work.

### PR DESCRIPTION
**Pull request by JakeWharton**

### CODE REVIEW

File: retrofit/src/main/java/retrofit2/Utils.java [hunk 1]
```diff
@@ -82,7 +82,9 @@ static boolean equals(Type a, Type b) {
       if (!(b instanceof ParameterizedType)) return false;
       ParameterizedType pa = (ParameterizedType) a;
       ParameterizedType pb = (ParameterizedType) b;
-      return equal(pa.getOwnerType(), pb.getOwnerType())
+      Object ownerA = pa.getOwnerType();
+      Object ownerB = pb.getOwnerType();
+      return (ownerA == ownerB || (ownerA != null && ownerA.equals(ownerB)))
```
Comment (NightlyNexus) [inline retrofit/src/main/java/retrofit2/Utils.java]: could be `equals(pa.getOwnerType(), pb.getOwnerType())` (one character change).
also avoids Type.equals usage, though i highly doubt there would ever be real case of a broken owner type.

File: retrofit/src/main/java/retrofit2/OkHttpCall.java [hunk 2]
```diff
@@ -110,16 +110,20 @@
...
         } catch (Throwable e) {
           callFailure(e);
           return;
         }
-        callSuccess(response);
+
+        try {
+          callback.onResponse(OkHttpCall.this, response);
+        } catch (Throwable t) {
+          t.printStackTrace();
```
Comment (swankjesse) [inline retrofit/src/main/java/retrofit2/OkHttpCall.java]: this doesn’t seem right

Comment (swankjesse) [reply retrofit/src/main/java/retrofit2/OkHttpCall.java]: oh, but it’s what we’ve always done? Sigh

Comment (JakeWharton) [reply retrofit/src/main/java/retrofit2/OkHttpCall.java]: Same as OkHttp. When the callback fails we have nowhere to go.

**Pull request merged**